build/extract_sources.py

```diff
@@ -170,7 +153,7 @@ def parse_args() -> argparse.Namespace:
     )
     parser.add_argument(
         "--buck2",
-        default="buck2",
+        default="",
         help="'buck2' command to use",
```

Comment:
Please update this to describe the behavior when a blank value is provided, now that it's more subtle.